plugins/modules/packaging/os/portage.py

```diff
@@ -506,7 +515,12 @@ def main():
     )
 
     module.emerge_path = module.get_bin_path('emerge', required=True)
-    module.equery_path = module.get_bin_path('equery', required=True)
+    try:
+        import portage
+        module.vartree = portage.vartree()
```

Comment:
Please don't modify `module`, but instead pass `vartree` and `equery_path` on as explicit function parameters.

Reply:
Also, this line is probably unlikely to cause an `ImportError` and shouldn't be inside the try-block anyway (it could be moved to an else-block, though).